ext/AvizExt/viz/spatial.jl

```diff
@@ -215,6 +216,127 @@ function ADRIA.viz.map!(
     )
 end
 
+"""
+    ADRIA.viz.connectivity(dom::Domain, network::SimpleWeightedDiGraph, conn_weights::AbstractVector{<:Real}; opts::Dict=Dict(), fig_opts::Dict=Dict(), axis_opts::Dict=Dict()) 
+    ADRIA.viz.connectivity!(g::Union{GridLayout, GridPosition}, dom::Domain,  network::SimpleWeightedDiGraph, conn_weights::AbstractVector{<:Real}; opts::Dict=Dict(), axis_opts::Dict=Dict()) 
+
+Produce visualisation of connectivity between reef sites with node size and edge visability
```

Comment:
Standardize to US English and fix typo

```suggestion
Produce visualization of connectivity between reef sites with node size and edge visibility
```